share/migrations/0043_merge_20170623_1531.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
You can avoid making merge migrations by deleted your old one and letting django recreate it.